data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants.py

```diff
@@ -14,7 +14,7 @@
 
 # Project imports
 from cdr_cleaner.cleaning_rules import sandbox_and_remove_pids
-from notebooks import bq
+import bq_utils
```

Comment:
Why did this change from the python big query client library implementation to bq_utils?  What pylint issue was this trying to fix?  This is a patch with working unit tests and the following pylint results.    
[closer_to_original_patch.txt](https://github.com/all-of-us/curation/files/4330380/closer_to_original_patch.txt)
```
(py3_env) [Fri Mar 13 10:42:53] [~/Projects/curation] lbiggers $ pylint --rcfile=.pylintrc data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants.py data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids.py tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants_test.py tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids_test.py 
************* Module data_steward.cdr_cleaner.cleaning_rules.sandbox_and_remove_pids
data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids.py:50:0: W0612: Unused variable 'get_sandbox_queries' (unused-variable)
data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids.py:79:0: W0612: Unused variable 'get_remove_pids_queries' (unused-variable)
************* Module tests.unit_tests.data_steward.cdr_cleaner.cleaning_rules.remove_aian_participants_test
tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants_test.py:67:8: W1505: Using deprecated method assertEquals() (deprecated-method)
tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants_test.py:13:0: W0612: Unused variable 'RemoveAIANParticipantsTest' (unused-variable)
************* Module tests.unit_tests.data_steward.cdr_cleaner.cleaning_rules.sandbox_and_remove_pids_test
tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids_test.py:88:8: W1505: Using deprecated method assertEquals() (deprecated-method)
tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids_test.py:116:8: W1505: Using deprecated method assertEquals() (deprecated-method)
tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids_test.py:15:0: W0612: Unused variable 'SandboxAndRemovePidsTest' (unused-variable)

-----------------------------------
Your code has been rated at 9.40/10

```
Are any of these messages build breaking messages?

Reply:
these are not build-breaking for messages for now. we are only checking for errors right now.